drizzlepac/haputils/align_utils.py

```diff
@@ -1352,16 +1352,16 @@ def update_image_wcs_info(tweakwcs_output, headerlet_filenames=None, fit_label=N
 
             # adds rel keywords if they don't exist
             keys_to_add = [
-                ("RELGEOM", "fitgeom for relative fit"),
-                ("RELMATCH", "number of matches for relative fit"),
-                ("RELRMS_D", "RMS in DEC of relative WCS fit(mas)"),
-                ("RELRMS_R", "RMS in RA of relative WCS fit(mas)"),
-                ("RELREFIM", "base reference image rootname for relative fit"),
+                ("RELGEOM", "N/A", "fitgeom for relative fit"),
+                ("RELMATCH", 0, "number of matches for relative fit"),
+                ("RELRMS_D", 0, "RMS in DEC of relative WCS fit(mas)"),
```

Comment:
Lines 1357 and 1358 are float (0.0).  Sorry to be a stickler, but the details can be important.

Reply:
Fixed.